.github/workflows/ansible-test.yml

```diff
@@ -104,13 +104,13 @@ jobs:
           python-version: ${{ matrix.python_version }}
 
       - name: Install community.general
-        run: ansible-galaxy collection install community.general -p ../../
+        run: 'for i in {1..5}; do [ "${i}" -ne 1 ] && sleep 5; ansible-galaxy collection install community.general -p ../../ && break; done'
 
       - name: Install community.crypto
-        run: ansible-galaxy collection install community.crypto -p ../../
+        run: 'for i in {1..5}; do [ "${i}" -ne 1 ] && sleep 5; ansible-galaxy collection install community.crypto -p ../../ && break; done'
 
       - name: Install ansible-base (${{ matrix.ansible_version }})
-        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible_version }}.tar.gz --disable-pip-version-check
+        run: 'for i in {1..5}; do [ "${i}" -ne 1 ] && sleep 5; pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible_version }}.tar.gz --disable-pip-version-check && break; done'
```

Comment:
`ansible-base` should be installed *before* the collections, that way if we improve `ansible-galaxy collection install` you will get the benefit.